protocol/validation/tx.go

```diff
@@ -227,7 +227,7 @@ func CheckTxWellFormed(tx *bc.Tx) error {
 			} else {
 				program = input.ControlProgram()
 			}
-			scriptStr, _ := vm.Disassemble(program)
+			scriptStr, _ := vm.Disassemble(program) // #nosec
```

Comment:
I tried setting `scriptStr` to the disassembly error in this case. I think that's probably the most helpful.